### PR TITLE
Update tempto to 1.36

### PR DESCRIPTION
**Pull request by kokosing**
Update tempto to 1.36
    
    Changes:
     - using sf1 scale for TPCH (instead of tiny). Result files were
     generated by reverting this commit (417622c).
     - inject statistics into hive for TPCH and TPCDS tables.
     Stats are not injected when kerberos is used for Hive.
    
    Motivation behind using sf1:
     - expected better behaviour of cost based optimizer rules, since
     differences in sizes of tables are bigger
     - do not hide memory related issues. General expectancy (rule) is that sf1
     TPCH queries should pass on Presto with Xmx=2G. There might be valid
     exception to this, but IMO it has to be understood why given such
     exceptional query consumes so much memory. Anyway, today all queries
     passes, so when something change then we will get regression.
    
    Cons of using sf1:
     - longer product-test travis job, now it 10 minutes more (40 min in total).

### CODE REVIEW

File: presto-product-tests/conf/docker/singlenode-hdfs-impersonation/docker-compose.yml [hunk 1]
```diff
@@ -4,3 +4,8 @@ services:
   presto-master:
     volumes:
       - ../../../conf/presto/etc/environment-specific-catalogs/singlenode-hdfs-impersonation/hive.properties:/docker/volumes/conf/presto/etc/catalog/hive.properties
+
+  application-runner:
```
Comment (losipiuk) [inline presto-product-tests/conf/docker/singlenode-hdfs-impersonation/docker-compose.yml]: Why this is special case to be tested without stats?
What about tests for accessing kerberized clusters?

Comment (kokosing) [reply presto-product-tests/conf/docker/singlenode-hdfs-impersonation/docker-compose.yml]: Sorry about that, but please consider this PR as WIP. This one failed without this. I have to check others as well as the ones in presto-checks.

**Comment by kokosing** [comment]
@losipiuk @electrum It is ready to be reviewed. Would you like to take a look?

File: presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestHiveTableStatistics.java [hunk 2]
```diff
@@ -241,6 +201,7 @@ public void testStatisticsForPartitionedTable()
                 row(null, null, null, null, 5.0));
     }
 
+    // This covers also stats calculation for unparitioned table
```
Comment (electrum) [inline presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestHiveTableStatistics.java]: Typo "unparitioned"
```java
// this also covers stats computation for unpartitioned tables

**Comment by electrum** [comment]
The increased Travis runtime is concerning, but there are two other jobs which also take 40 minutes, so it doesn't increase the total runtime.

File: presto-product-tests/src/main/resources/tempto-configuration.yaml [hunk 3]
```diff
@@ -20,8 +20,11 @@ databases:
     table_manager_type: hive
     warehouse_directory_path: /user/hive/warehouse
     path: /user/hive/warehouse
-    analyze_immutable_tables: false
-    analyze_mutable_tables: false
+    inject_stats_for_immutable_tables: true
+    inject_stats_for_mutable_tables: true
+    metastore:
```
Comment (electrum) [inline presto-product-tests/src/main/resources/tempto-configuration.yaml]: Curious, why was this not needed before? Was it hard-coded in Tempto?

Comment (kokosing) [reply presto-product-tests/src/main/resources/tempto-configuration.yaml]: There was no such things in previous version of tempto. it is used to inject data statistics into hive metastore (previously we tried to run `ANALYZE`, but it was not a good idea...)

**Comment by kokosing** [comment]
@losipiuk Are you ok with this?

**Pull request merged**